src/runtime/resource/detail/detail_partitioner.cpp

```diff
@@ -505,7 +505,6 @@ namespace hpx { namespace resource { namespace detail
         affinity_data_.set_num_threads(new_pu_nums.size());
         affinity_data_.set_pu_nums(std::move(new_pu_nums));
         affinity_data_.set_affinity_masks(std::move(new_affinity_masks));
-        affinity_data_.init_cached_pu_nums(new_pu_nums.size());
```

Comment:
What is the rationale of this change? is that related to the clang-tidy checks?

Reply:
Yes, `new_pu_nums` was moved a few lines above. When looking at the code of `affinity_data`, `init_cached_pu_nums` does nothing in this specific case anyway: https://github.com/STEllAR-GROUP/hpx/blob/master/src/runtime/threads/policies/affinity_data.cpp#L324